src/content/tutorials/observability/data-ingestion/metrics/_index.md

```diff
@@ -68,14 +68,14 @@ spec:
 
 No matter if you are using Helm Charts or GitOps and Kustomize, just put the ServiceMonitor CR next to your app and apply it in the same way. Once it's applied you can check either the _ServiceMonitors Overview_ or _ServiceMonitors Details_ dashboards, or just search for the new metrics ingested in your installation's Grafana to make sure that your containers are being scraped by the new monitoring agents.
 
-__Warning:__ The ServiceMonitor needs to be labeled with `application.giantswarm.io/team: <YOUR-TEAM-NAME>` for the metrics agent to be able to discover it and start collecting metrics.
+__Warning:__ The ServiceMonitor needs to be labeled with `observability.giantswarm.io/tenant: <YOUR-TENANT-NAME>` for the metrics agent to be able to discover it and start collecting metrics.
 
 ## ServiceMonitor vs. PodMonitor
 
 In most cases, a ServiceMonitor should cover most of your monitoring use cases but it can happen on rare occasions that a container doesn't need a service to run and it doesn't make sense to create one just for the sake of monitoring it. That's when the PodMonitor comes into action. You can find a few other examples where PodMonitor makes sense [in this discussion](https://github.com/prometheus-operator/prometheus-operator/issues/3119) in the Prometheus Operator Project.
```

Comment:
And I think we need to add a link to the podlogs?

Reply:
I saw that in the issue but I'm not sure what link we're trying to make between those 2

Reply:
I think the goal is more for navigation purposes,  like if you want to also collect logs, go there kind of thing

Reply:
Oh ok

Reply:
:heavy_check_mark: